src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,79 @@
+package lotto.domain;
+
+import lotto.exception.ErrorMessage;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+// 로또
+public class Lotto {
+
+    private List<Integer> numbers;
```

Comment:
로또 번호 숫자 하나를 의미하는 클래스를 도출해보는 것은 어떨까요 ?
이미 사용되고 있는 LottoNumber가 적절해 보이기도하는데.. 고민해볼 포인트인 것 같아요 🤔 

LottoNumber클래스를 사용하게되면 1~45 사이의 숫자라는 것을 믿고 신뢰하고 사용할 수 있습니다
아래 numberRangeValid() 검증 또한 Lotto에서 하지 않아도 되지요 :)

추가적으로 일급 컬렉션을 사용하실때는 final을 사용하면 좀 재할당을 막아 좀 더 불변에 가깝게 사용할 수 있습니다.

```suggestion
    private final List<Integer> numbers;
```